src/buttons/Button.tsx

```diff
@@ -133,6 +135,21 @@ const Button: RneFunctionComponent<ButtonProps> = (props) => {
     busy: !!loading,
   };
 
+  const getOrientation = () => {
+    switch (iconPosition) {
+      case 'top':
+        [styles.buttonOrientation, { flexDirection: 'column' }];
+        break;
+      case 'bottom':
+        [styles.buttonOrientation, { flexDirection: 'column-reverse' }];
+        break;
+      case 'right':
+        [styles.buttonOrientation, { flexDirection: 'row-reverse' }];
+        break;
+      default:
+        [styles.buttonOrientation, { flexDirection: 'row' }];
+    }
+  };
```

Comment:
We can simply use object 
let iconStyle={top:"column",...other positions...}[iconPosition] || "column";